configure.py

```diff
@@ -1013,8 +1013,15 @@ def configure_node(o):
   else:
     o['variables']['node_use_dtrace'] = 'false'
 
-  use_large_pages = (flavor == 'linux' and options.node_use_large_pages)
-  o['variables']['node_use_large_pages'] = b(use_large_pages)
+  if options.node_use_large_pages:
```

Comment:
I think this can be simplified to
```python
if options.node_use_large_pages and flavor != 'linux':
  raise Exception(
    'Large pages are supported only on Linux Systems.')
o['variables']['node_use_large_pages'] = b(options.node_use_large_pages)
```
wdyt?

Reply:
@lundibundi There are just so many ways. This is makes it look succinct for sure. I'll make the change.

Reply:
@lundibundi Pushed new commit addressing both the suggestions. Thanks.